clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java

```diff
@@ -102,20 +102,20 @@ public void testResolveUnknownHostException() throws UnknownHostException {
 
     @Test
     public void testResolveDnsLookup() throws UnknownHostException {
-        // Note that kafka.apache.org resolves to 2 IP addresses
+        // Note that kafka.apache.org resolves to at least 2 IP addresses
```

Comment:
Comment here and below refer to 2 IP addresses still.

Reply:
Are you saying to change it back? I thought it was fair to say "at least 2" to reflect we are asserting greater than 1.